--- a/debian/control
+++ b/debian/control
@@ -5,7 +5,7 @@ Priority: optional
 # clearwater-infrastructure explicitly checks for packages of this name when
 # updating
 Maintainer: Project Clearwater Maintainers <maintainers@projectclearwater.org>
-Build-Depends: debhelper (>= 8.0.0), cassandra (= 2.1.15), openjdk-7-jdk
+Build-Depends: debhelper (>= 8.0.0), cassandra (= 2.1.21), openjdk-7-jdk
 Standards-Version: 3.9.2
 Homepage: http://projectclearwater.org/
