--- a/bono/Dockerfile
+++ b/bono/Dockerfile
@@ -1,10 +1,13 @@
 FROM clearwater/base
 MAINTAINER maintainers@projectclearwater.org

-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes bono
+RUN mkdir /tmp/clearwater-extra-debs
+ADD clearwater-debs /tmp/clearwater-extra-debs
+
+RUN dpkg -i /tmp/clearwater-extra-debs/*; apt-get install -f -y; dpkg -i /tmp/clearwater-extra-debs/*;
+
 RUN sed -e 's/\(echo 0 > \/proc\/sys\/kernel\/yama\/ptrace_scope\)/# \0/g' -i /etc/init.d/bono

-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes restund
 RUN sed -e 's/\(echo 0 > \/proc\/sys\/kernel\/yama\/ptrace_scope\)/# \0/g' -i /etc/init.d/restund

 COPY bono.supervisord.conf /etc/supervisor/conf.d/bono.conf
