--- a/chronos/Dockerfile
+++ b/chronos/Dockerfile
@@ -1,7 +1,10 @@
 FROM clearwater/base
 MAINTAINER maintainers@projectclearwater.org

-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes chronos
+RUN mkdir /tmp/clearwater-extra-debs
+ADD clearwater-debs /tmp/clearwater-extra-debs
+
+RUN dpkg -i /tmp/clearwater-extra-debs/*; apt-get install -f -y

 COPY snmpd.supervisord.conf /etc/supervisor/conf.d/snmpd.conf
 COPY chronos.supervisord.conf /etc/supervisor/conf.d/chronos.conf
