--- sprout/Dockerfile
+++ sprout/Dockerfile
@@ -1,11 +1,18 @@
 FROM clearwater/base
 MAINTAINER maintainers@projectclearwater.org
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes sprout
+RUN apt-get install ruby-dev -y && gem install ffi -v 1.9.5
+
+RUN mkdir /tmp/clearwater-extra-debs
+ADD clearwater-debs /tmp/clearwater-extra-debs
+
+RUN dpkg -i /tmp/clearwater-extra-debs/*; apt-get install -f -y; dpkg -i /tmp/clearwater-extra-debs/*
 RUN sed -e 's/\(echo 0 > \/proc\/sys\/kernel\/yama\/ptrace_scope\)/# \0/g' -i /etc/init.d/sprout
-RUN apt-get install -y --force-yes clearwater-snmp-handler-astaire
 #RUN echo "log_level=5" >> /etc/clearwater/user_settings
 
+RUN echo "num_http_threads=50" > /etc/clearwater/user_settings
+RUN echo "num_worker_threads=50" >> /etc/clearwater/user_settings
+
 COPY sprout.supervisord.conf /etc/supervisor/conf.d/sprout.conf
 COPY clearwater-group.supervisord.conf /etc/supervisor/conf.d/clearwater-group.conf
 
